### PR TITLE
Add tmux-clima as another module for weather

### PR DESCRIPTION
**Pull request by xafarr**
#160

### CODE REVIEW

**Comment by vdbe** [comment]
What does this plugin provide that the the current weather module doesn't?
It seems to also use OpenWeatherMap so the result are the same I guess (and you need your own api key).

Before merging this it needs documentation in the README.

**Comment by xafarr** [comment]
It does the same thing but in a different way. Also, [tmux-weather](https://github.com/xamut/tmux-weather) is pretty inactive. And [tmux-clima](https://github.com/vascomfnunes/tmux-clima) can be an extra option for people to use with **catppuccin/tmux**. Updated the PR with README update. Thanks.

File: README.md [hunk 1]
```diff
@@ -363,15 +363,22 @@ Add the cpu module to the status modules list.
 set -g @catppuccin_status_modules_right "... cpu ..."
 ```
 
-### Weather module
+### Weather modules
```
Comment (vdbe) [inline README.md:366]: Link under `## Content` needs to be updated

File: README.md [hunk 2]
```diff
@@ -363,15 +363,22 @@ Add the cpu module to the status modules list.
...
 
-#### Requirements
+There are a couple of modules that can be used to retrieve weather data.
+
+- [tmux-weather](https://github.com/xamut/tmux-weather)
+- [tmux-clima](https://github.com/vascomfnunes/tmux-clima)
+
+#### [tmux-weather](https://github.com/xamut/tmux-weather)
```
Comment (vdbe) [inline README.md:373]:
```suggestion
#### tmux-weather
```
Already linked 3 lines below

File: README.md [hunk 3]
```diff
@@ -384,6 +391,27 @@ Add the weather module to the status modules list.
 set -g @catppuccin_status_modules_right "... weather ..."
 ```
 
+#### [tmux-clima](https://github.com/vascomfnunes/tmux-clima)
```
Comment (vdbe) [inline README.md:394]:
```suggestion
#### https://github.com/vascomfnunes/tmux-clima
```

**Review by vdbe — CHANGES_REQUESTED** [review]

**Comment by vdbe** [comment]
It currently does not do anything extra except set prefix+W.
I will leave this open in case this changes and for further discussion/requests.

**Comment by xafarr** [comment]
I have been using **tmux-clima** for almost 2 years and contribute to it as well. I also like **catppuccin** project and use it in all terminal programs. I raised this PR for people like me to have an option to use these together. I used to use **tmux-weather** but was constantly having problems with *wttr.in*. That was the reason I moved to this plugin. *OpenWeatherMap API* is pretty stable and gives free 1000 calls per day.

**Review by vdbe — APPROVED** [review]

**Pull request merged**